Example/Tests/Tests.m

```diff
@@ -0,0 +1,28 @@
+/*
+ Copyright 2012 Javier Soto (ios@javisoto.es)
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+SpecBegin(InitialSpecs)
+
+// This spec was auto-generated by cocoapods.
+
+describe(@"Passing Spec", ^{
+    
+    it(@"can do maths", ^{
+        expect(1).beLessThan(42);
+    });
+});
+
```

Comment:
I think it could be removed. Not sure how easy/difficult it'd be to test core graphics logic, but I guess some tests could be written to ensure that the configurability of the library is correct. Perhaps adding unit tests/coverage/coveralls would best belong in a separate github issue though?